lib/docker/executeDockerBuild.ts

```diff
@@ -113,6 +119,7 @@ export function executeDockerBuild(options: DockerOptions): ExecuteGoal {
         const { goalEvent, context, project } = gi;
 
         const optsToUse = mergeOptions<DockerOptions>(options, {}, "docker.build");
+        optsToUse.builderPath = (optsToUse.builderPath) ? optsToUse.builderPath : ".";
```

Comment:
If I recall correctly, setting of defaults happens in DockerBuild goal. Could be move this to there?

Reply:
Done

Reply:
I even didn't rebase the commit.